### PR TITLE
Electronic Book Support (FR)

### PR DESCRIPTION
**Pull request by Marc66**

### CODE REVIEW

**Comment by Marc66** [comment]
Added French language for the Web site

File: Translations/fr.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -57,7 +57,7 @@
 "Safari extension" = "Extension Safari";
 "ACE Archive" = "Archive ACE";
 "XIP Secure Archive" = "Archive sécurisée XIP";
-"Electronic Book" = "Electronic Book";
+"Electronic Book" = "Livre électronique";
 "Apple Archive" = "Apple Archive";
```
Comment (aonez) [inline Translations/fr.lproj/InfoPlist.strings:61]: This one is translated in the bundled macOS extractor tool so might as well translate it 😛

**Review by aonez — CHANGES_REQUESTED** [review]

**Comment by aonez** [comment]
Already added the web French translation. Will fix the `l\'` string. You can't use scape inside `' '` strings but you can use scape inside `" "`.

**Comment by Marc66** [comment]
> Already added the web French translation. Will fix the `l\'` string. You can't use scape inside `' '` strings but you can use scape inside `" "`.

Thank you, looking good. Sorry for the \' string, I couldn't test while I was translating

**Pull request merged**